bakerydemo/static/css/main.css

```diff
@@ -291,181 +382,229 @@ nav {
   margin: 15px 0 5px;
   display: none;
 }
+
 #main-navigation {
   padding-left: 0;
 }
+
 @media (min-width: 768px) {
   nav {
     margin: 15px 0 0 0;
-    border-top: 1px solid rgba(255,255,255,0.1);
+    border-top: 1px solid rgba(255, 255, 255, 0.1);
     display: block;
   }
 }
-.nav-pills>li+li {
+
+.nav-pills > li + li {
   margin-left: 0;
 }
-.nav-pills>li>a {
+
+.nav-pills > li > a {
   border-radius: 0;
   border-top: 1px solid transparent;
   color: white;
-  color: white;
-  font-family: 'Lato', sans-serif;
+  font-family: "Open Sans", sans-serif;
   font-size: 11px;
   font-weight: 300;
   letter-spacing: 0.15em;
   margin-top: -1px;
   padding: 10px 10px;
   text-transform: uppercase;
 }
+
 @media (min-width: 768px) {
-  .nav-pills>li>a {
+  .nav-pills > li > a {
     padding: 10px 20px;
     font-size: 14px;
   }
 }
-.nav-pills>li>a, .nav-pills>li>a:focus, .nav-pills>li>a:hover,
-.nav-pills>li, .nav-pills>li, .nav>li>a, .nav>li>a {
+
+.nav-pills > li > a,
+.nav-pills > li > a:focus,
+.nav-pills > li > a:hover,
+.nav-pills > li,
+.nav-pills > li,
+.nav > li > a,
+.nav > li > a {
   border-top: 1px solid transparent;
 }
+
 /* The following is to stop a pixel shift on hover */
-.nav-pills>.breads {
+.nav-pills > .breads {
   width: 90px;
 }
-.nav-pills>.locations {
+
+.nav-pills > .locations {
   width: 140px;
 }
-.nav-pills>.blog {
+
+.nav-pills > .blog {
   width: 86px;
 }
-.nav-pills>.gallery {
+
+.nav-pills > .gallery {
   width: 115px;
 }
-.nav-pills>.contactus {
+
+.nav-pills > .contactus {
   width: 148px;
 }
-.nav-pills>.about {
+
+.nav-pills > .about {
   width: 88px;
 }
-.nav-pills>li.active>a, .nav-pills>li.active>a:focus, .nav-pills>li.active>a:hover,
-.nav-pills>li.active, .nav-pills>li:hover, .nav>li>a:focus, .nav>li>a:hover {
+
+.nav-pills > li.active > a,
+.nav-pills > li.active > a:focus,
+.nav-pills > li.active > a:hover,
+.nav-pills > li.active,
+.nav-pills > li:hover,
+.nav > li > a:focus,
+.nav > li > a:hover {
   color: #d4566b;
   background-color: transparent;
   border-top: 1px solid #d4566b;
   font-weight: 400;
 }
-.nav-pills> li:first-of-type > a {
+
+.nav-pills > li:first-of-type > a {
   padding-left: 0;
 }
-.nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
+
+.nav .open > a,
+.nav .open > a:focus,
+.nav .open > a:hover {
   background-color: transparent;
   border-top: 1px solid #d4566b;
 }
 
 .dropdown-menu {
-  background-color: rgba(25,17,18,1);
+  background-color: rgba(25, 17, 18, 1);
   border-radius: 0;
   border: transparent;
 }
 
-.dropdown-menu > li> a {
+.dropdown-menu > li > a {
   border-bottom: 1px solid #371c19;
   color: #fff;
-  font-family: 'Lato', sans-serif;
+  font-family: "Open Sans", sans-serif;
```

Comment:
suggestion: assign the fonts to variables which will make it easier to update should they need to be changed

Reply:
Thanks, added this in